qiskit/providers/aer/noise/errors/standard_errors.py

```diff
@@ -66,14 +76,20 @@ def mixed_unitary_error(noise_ops, standard_gates=True):
 
     Args:
         noise_ops (list[pair[matrix, double]]): unitary error matrices.
-        standard_gates (bool): Check if input matrices are standard gates.
+        standard_gates (bool): DEPRECATED, Check if input matrices are standard gates.
 
     Returns:
         QuantumError: The quantum error object.
 
     Raises:
         NoiseError: if error parameters are invalid.
     """
+    if standard_gates:
```

Comment:
IT would probably be better to deprecate this whole function. There isn't any reason for it when you can initialize QuantumError directly from the same noise ops.

Reply:
I think it's still useful to create mixed unitary error from matrices, e.g.
```
mixed_unitary_error([(mat1, p1), (mat2, p2)])
```
That is a bit simpler than
```
QuantumError([(UnitaryGate(mat1), p1), (UnitaryGate(mat2), p1)])
```
And if we were removing `mixed_unitary_error`, we should remove `kraus_error` as well because `kraus_error([mat1, mat2])` can be replaced with  equivalent with `QuantumError(Kraus([mat1, mat2])`.